### DISCUSSION TITLE
Does this work on Win 10 latest builds 21H1/21H2?

### DISCUSSION THREAD

**Discussion by Globespy** [Codectory/AutoActions#97] [Q&A]
Does this program work on Win 10 21H1/21H2?
Or is it only for those running Win 10 test versions that had Auto-HDR?

Thanks.

**Comment by Codectory** [comment]
AutoActions is not related to the Auto-HDR feature of windows. AutoActions
will run actions based on the running apps, e.g changing resolution or
enabling HDR ins Windows options. Auto-HDR of Windows calculates SDR
content to look like HDR content.
So, yes so , you can use it with any Windows 1” version.

Globespy ***@***.***> schrieb am Fr. 21. Jan. 2022 um 02:35:
